### DISCUSSION TITLE
Rendered Rust Book example

### DISCUSSION THREAD

**Discussion by afranke** [max-heller/mdbook-pandoc#121] [Q&A] [answered]
The README has a link to a rendered PDF of the Rust Book. This ouput doesn’t very LaTeX-y, unlike e.g. the Cargo Book one. How was this result achieved?

Also, the Ferris images on page 8 don’t appear. Not sure whose fault it is, so I didn’t file an issue for it.

**Comment by max-heller** [comment, marked as answer]
Hi Alexandre, the sample books are rendered by `#[test]`s in `lib.rs` -- in particular, this one for the Rust book:
https://github.com/max-heller/mdbook-pandoc/blob/24b1beb60adb69297e6f69fdcd1542297d9dfebe/src/lib.rs#L1596-L1607

`Config::pdf()` mainly just configures fonts and page margins, which may be what makes it not look very "LaTeX-y":
https://github.com/max-heller/mdbook-pandoc/blob/24b1beb60adb69297e6f69fdcd1542297d9dfebe/src/lib.rs#L512-L540

It probably makes sense to point to this config from the README but I wasn't sure of the best way to do so without having the link/config go stale eventually.

**Reply by afranke** [reply]
> `Config::pdf()` mainly just configures fonts and page margins, which may be what makes it not look very "LaTeX-y":

Indeed! The fonts and margins are usually what gives away the use of LaTeX. Thanks.

**Comment by max-heller** [comment]
> unlike e.g. the Cargo Book one

Could you elaborate on this?

**Reply by afranke** [reply]
Huh. I don’t know what I was looking at yesterday, but the Cargo book I just downloaded now also has that styling. Sorry about the confusion.

**Comment by max-heller** [comment]
> Also, the Ferris images on page 8 don’t appear.

I think the issue here is that the Ferris images are in `<img>` tags, which pandoc doesn't parse as an image from markdown. This will need a preprocessing pass to convert them to Markdown's `![]()` image syntax

**Reply by max-heller** [reply]
Implemented in #124!
